### DISCUSSION TITLE
Cannot find module 'express', Did you forget to add it to "dependencies" in `package.json`?

### DISCUSSION THREAD

**Discussion by code-with-shahzad** [vercel/community#6438] [Help] [answered]
### Page to Investigate

runtime logs

### Steps to Reproduce

I have include everything inside package.json but still error what is the problem 

Cannot find module 'express'
Require stack:
- /var/task/index.js
- /var/task/___now_launcher.js
- /var/runtime/index.mjs
Did you forget to add it to "dependencies" in `package.json`?
INIT_REPORT Init Duration: 82.09 ms	Phase: init	Status: error	Error Type: Runtime.ExitError
Cannot find module 'express'
Require stack:
- /var/task/index.js
- /var/task/___now_launcher.js
- /var/runtime/index.mjs
Did you forget to add it to "dependencies" in `package.json`?
INIT_REPORT Init Duration: 135.19 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError
Error: Runtime exited with error: exit status 1

{
  "name": "writebolt-backend",
  "version": "1.0.0",
  "description": "writebolt backend",
  "main": "index.js",
  "scripts": {
    "nodemon": "nodemon index.js"
  },
  "dependencies": {
    "axios": "^1.6.8",
    "body-parser": "^1.20.2",
    "cors": "^2.8.5",
    "dotenv": "^16.4.5",
    "express": "^4.18.2",
    "nodemon": "^3.1.0",
    "xmldom": "^0.6.0"
  }
}

**Comment by amyegan** [comment, marked as answer]
Hey @code-with-shahzad. Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example)?

I have a small [Express example repo](https://github.com/amyegan/express-vercel) based on the [Using Express.js with Vercel](https://vercel.com/guides/using-express-with-vercel) guide. But I wasn't able to reproduce this error with it.

**Reply by code-with-shahzad** [reply]
Thanks amyegan let me check if I resolve this error by following your file structure

**Reply by code-with-shahzad** [reply]
Thank you so much error resolve @amyegan
